src/server/api/routers/recaptcha.ts

```diff
@@ -0,0 +1,55 @@
+import { createTRPCRouter, protectedProcedure } from "../trpc.js";
+import { TRPCError } from "@trpc/server";
+import { z } from "zod";
+import { env } from "~/env.mjs";
+import { type RecaptchaResponse } from "~/types/recaptcha";
+
+const SCORE_THRESHOLD = 0.5;
+export const recaptchaRouter = createTRPCRouter({
+  verify: protectedProcedure
+    .input(z.object({ captchaToken: z.string() }))
+    .mutation(async ({ input }) => {
+      const { captchaToken } = input;
+
+      // Check if it's not in production (development mode)
+      if (env.NODE_ENV !== "production") {
```

Comment:
Let's add staging here also, so we can demo it eventually.

Reply:
I believe this will suffice ? 
`      // Check if it's (development mode)
      if (env.NODE_ENV === "development") {
        return false;
      }`